### PR TITLE
Whittier health scraper

### PR DESCRIPTION
**Pull request by pfurbacher**
Had a bit of luck by finding another provider, Butler County, Ohio, General Health District (BCGHD) who has the same SignUpGenius calendar style. They had a stale calendar, but it had availability. Pretty straightforward after that. (The BCGHD page, stripped of scripts, serves as a test resource.)

### CODE REVIEW

File: no-browser-site-scrapers/WhittierHealth/index.js [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            // Add all day objects to results.availability
+            monthAvailabilityMap.forEach((value, key) => {
+                availabilityContainer.availability[key] = {
+                    numberAvailableAppointments: value,
+                    hasAvailability: !!value,
+                };
+            });
+
+            hasAvailability ||=
+                Object.keys(availabilityContainer.availability).length > 0;
```
Comment (harcod) [inline no-browser-site-scrapers/WhittierHealth/index.js:69]: I am getting an "invalid token '||='" error when attempting to run this.  I'm running node.js v14.15.5.  According to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment), this construct isn't fully supported until node.js v15. (which we are requiring yet).

package.json sets the minimum requirement at `        "node": ">= 14.0.0",`

Comment (pfurbacher) [reply no-browser-site-scrapers/WhittierHealth/index.js:69]: Ok, I'll fix that. Also see about downgrading to 14.x

Comment (pfurbacher) [reply no-browser-site-scrapers/WhittierHealth/index.js]: Committed and downgrade done.

**Review by harcod — APPROVED** [review]

**Review by iann — APPROVED** [review]
LGTM. nice and quick.

**Pull request merged**